flopy/mf6/utils/createpackages.py

```diff
@@ -3,6 +3,8 @@
 import datetime
 import textwrap
 from enum import Enum
+
+# keep below as absolute imports
```

Comment:
Nice find.  This was the only one I could think of that shouldn't be relative.

Reply:
Yes, that was my finding too. Scripts need to have absolute imports. It is run in modflow6's [autotest/update_flopy.py](https://github.com/MODFLOW-USGS/modflow6/blob/6b952a69559792c9425bcc1127d325168db16aaf/autotest/update_flopy.py#L60-L79)